### PR TITLE
Update AUTHORS.md

### LINKED ISSUE #260: list previous authors in AUTHORS.md

### PR DESCRIPTION
**Pull request by swzCuroverse**
Will fix #260 

Adding authors list for people with 5 or more contributions and also Peter who helped Bruno with the re-write and the original content of the User Guide.  I kept those people who had contributed to user guide content and not those people who were working on the software carpentry templates.

### CODE REVIEW

**Comment by elehcim** [comment]
thanks, my slug is "@elehcim"

**Comment by mr-c** [comment]
> Adding authors list for people with 5 or more contributions and also Peter who helped Bruno with the re-write and the original content of the User Guide. 

One contribution is enough, yeah. Doesn't hurt or cost us.

> I kept those people who had contributed to user guide content and not those people who were working on the software carpentry templates.

Okay, we should link to the template repo then

**Review by Smyja — COMMENTED** [review]
Oghenemaro Akpobi is my full name.

**Review by Smyja — CHANGES_REQUESTED** [review]
Oghenemaro Akpobi is my full name

**Comment by swzCuroverse** [comment]
> > Adding authors list for people with 5 or more contributions and also Peter who helped Bruno with the re-write and the original content of the User Guide.
> 
> One contribution is enough, yeah. Doesn't hurt or cost us.
> 
> > I kept those people who had contributed to user guide content and not those people who were working on the software carpentry templates.
> 
> Okay, we should link to the template repo then

Yes, I don't know that - so if someone could tell me that would be great.

**Comment by swzCuroverse** [comment]
> thanks, my slug is "@elehcim" , not @lehcim. Orcid ID https://orcid.org/0000-0002-6324-5713

Thanks - I have updated for this.

**Comment by swzCuroverse** [comment]
> Oghenemaro Akpobi is my full name

I have added this.

**Review by mr-c — CHANGES_REQUESTED** [review]
The formatting at https://github.com/common-workflow-language/user_guide/blob/swz-author-list/AUTHORS.md looks a bit off

also visible at https://github.com/common-workflow-language/user_guide/pull/283/files by selecting "Display the rich diff"
image

**Comment by Smyja** [comment]
> > Oghenemaro Akpobi is my full name
> 
> I have added this.

It's Akpobi not Akpo.

**Comment by Smyja** [comment]
> > > > Oghenemaro Akpobi is my full name
> > > 
> > > 
> > > I have added this.
> > 
> > 
> > It's Akpobi not Akpo.
> 
> Whoops, bad cut and paste. I will fox ASAP, sorry about that

Okay

**Comment by swzCuroverse** [comment]
> The formatting at https://github.com/common-workflow-language/user_guide/blob/swz-author-list/AUTHORS.md looks a bit off
> 
> also visible at https://github.com/common-workflow-language/user_guide/pull/283/files by selecting "Display the rich diff" image

This has been fixed.

File: AUTHORS.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+* Michele Mastropietro (@elehcim) https://orcid.org/0000-0002-6324-5713
+* Robin Long (@longr) 
+* Manabu Ishii (@manabuishii)
+* Michael Crusoe (@mr-c) http://orcid.org/0000-0002-2961-9670
+* @rmccole
+* Adrian Sharma (@sharmatime)
+* Oghenemaro Akpobi (@Smyja)
+* Stian Soiland-Reyes (@stain) [https://orcid.org/0000-0001-9842-9718
+   
+Thanks to:
```
Comment (mr-c) [inline AUTHORS.md:15]:
```suggestion
* Benjamin Carr (@BenjaminHCCarr) <https://orcid.org/0000-0002-6206-4638>
* Emannuel Fernandes de Oliveira Carvalho (@emannuelOC)
* Eugene Miloslavsky (@emiloslavsky)
* Henry Liu (@HenryLiu0)
* Ngumih Fien (@Fienne)
* @jessMaia
* Bruno P. Kinoshita (@kinow) <https://orcid.org/0000-0001-8250-4074>
* Konstantin Taletskiy (@ktaletsk) <https://orcid.org/0000-0001-6846-9815>
* Kushal Beniwal (@KushalBeniwal)
* Peter Amstutz (@tetron) <https://orcid.org/0000-0003-3566-7705>
* Toby Hodges (@tobyhodges) <https://orcid.org/0000-0003-1766-456X>
* Michele Mastropietro (@elehcim) <https://orcid.org/0000-0002-6324-5713>
* Robin Long (@longr) <https://orcid.org/0000-0003-2249-645X>
* Mark A. Jensen  (@majensen) <https://orcid.org/0000-0001-5215-101X>
* Manabu Ishii (@manabuishii) <https://orcid.org/0000-0002-5843-4712>
*  @medcelerate
* Michael Crusoe (@mr-c) <http://orcid.org/0000-0002-2961-9670>
* Ruth B. McCole  (@rmccole) <https://orcid.org/0000-0002-1861-7055>
* @sersorrel
* @swzCuroverse
* Adrian Sharma (@sharmatime)
* Oghenemaro Akpobi (@Smyja)
* Stian Soiland-Reyes (@stain) <https://orcid.org/0000-0001-9842-9718>
*  Daiki Tsuchiya (@tsuchi8) <https://orcid.org/0000-0003-2112-9316>
* Tomoya Tanjo (@tom-tan) <https://orcid.org/0000-0002-4421-9659>
* Wolfgang Gerlach (@wgerlach) <https://orcid.org/0000-0002-1480-1242>
* Zipho Mashologu (@zipho)

Thanks to:
```
rmccole's name is from her repo <https://github.com/rmccole/UCEs_genome_organization>

**Review by mr-c — CHANGES_REQUESTED** [review]

**Review by swzCuroverse — COMMENTED** [review]
This is Ok -- you could add my name if you were going to add my handle.

**Review by swzCuroverse — COMMENTED** [review]
this looks fine to me.  i will add my full name later.

File: AUTHORS.md (hunk 1 above)
Comment (swzCuroverse) [reply AUTHORS.md:15]: I can't review this cause it is against my own pull request for some reason.

**Review by tetron — DISMISSED** [review]

**Comment by swzCuroverse** [comment]
@mr-c is this good now?

File: AUTHORS.md [hunk 2]
```diff
@@ -1 +1,32 @@
-FIXME: list authors' names and email addresses.
\ No newline at end of file
+Thank you to the following contributors (in alphabetical order by user name):
```
Comment (kinow) [inline AUTHORS.md:1]: I think the order is not being followed in the doc.

Comment (mr-c) [reply AUTHORS.md:1]: Good catch, I have fixed that

**Review by mr-c — APPROVED** [review]

**Pull request merged**